tests/shared/PSPIntegrationTestDefs.js

```diff
@@ -194,598 +194,589 @@ gpii.tests.pspIntegration.testDefs = [
-                    func: "gpii.test.snapshotSettings",
-                    args: ["{tests}.options.data.initial.settingsHandlers", "{tests}.settingsStore", "{nameResolver}", "{testCaseHolder}.events.onSnapshotComplete.fire"]
-                }, {
-                    event: "{testCaseHolder}.events.onSnapshotComplete",
-                    listener: "fluid.identity"
-                }, {
-                    func: "{loginRequest}.send"
-                }, {
-                    event: "{loginRequest}.events.onComplete",
-                    listener: "gpii.test.loginRequestListen"
-                }, {
-                    func: "gpii.test.checkConfiguration",
-                    args: ["{tests}.options.data.initial.settingsHandlers", "{nameResolver}", "{testCaseHolder}.events.onCheckConfigurationComplete.fire"]
-                }, {
-                    event: "{testCaseHolder}.events.onCheckConfigurationComplete",
-                    listener: "fluid.identity"
-                }, {
-                    func: "{pspClient}.connect"
-                }, {
-                    event: "{pspClient}.events.onConnect",
-                    listener: "gpii.tests.pspIntegration.connectionSucceeded"
-                }, {
-                    event: "{pspClient}.events.onReceiveMessage",
-                    listener: "gpii.tests.pspIntegration.checkPayload",
-                    args: ["{arguments}.0", "modelChanged"]
-                }, {
-                    funcName: "gpii.tests.pspIntegration.sendMsg",
-                    args: [ "{pspClient}", [ "preferences","http://registry\\.gpii\\.net/common/magnification"], 3]
-                }, {
-                    event: "{pspClient}.events.onReceiveMessage",
-                    listener: "gpii.tests.pspIntegration.checkPayload",
-                    args: ["{arguments}.0", "preferencesApplied"]
-                }, {
-                    func: "gpii.test.checkConfiguration",
-                    args: ["{tests}.options.data.afterChangeMagnification.settingsHandlers", "{nameResolver}", "{testCaseHolder}.events.onCheckConfigurationComplete.fire"]
-                }, {
-                    event: "{testCaseHolder}.events.onCheckConfigurationComplete",
-                    listener: "fluid.identity"
-                }, {
-                    func: "{logoutRequest}.send"
-                }, {
-                    event: "{logoutRequest}.events.onComplete",
-                    listener: "gpii.test.logoutRequestListen"
-                }, {
-                    func: "gpii.test.checkRestoredConfiguration",
-                    args: ["{tests}.options.data.initial.settingsHandlers", "{tests}.settingsStore", "{nameResolver}", "{testCaseHolder}.events.onCheckRestoredConfigurationComplete.fire"]
-                }, {
-                    event: "{testCaseHolder}.events.onCheckRestoredConfigurationComplete",
-                    listener: "fluid.identity"
-                }
-            ]
+            {
```

Comment:
Any chance we could lose the massive formatting diffs in this file?

Reply:
As stated above, I really really dislike not indenting multi-element arrays, and indenting them more cleanly made it easier to do the review work required for this pull.  I think of it as something akin to alphabetising a previously unordered list of solutions registry entries.  It made it easier to perform the work in question today, and hopefully will make it easier to maintain things in the future, even if it does bloat the pull.

Reply:
In follow up work I have noticed additional small multi-element arrays and left them alone for now.